Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp

```diff
@@ -159,6 +162,27 @@ void ControllerInterface::UpdateInput()
 }
 
 //
+// RegisterDeviceCallback
+//
+// Register a callback to be called when there is a new device
+//
+void ControllerInterface::RegisterDeviceCallback(std::function<void()> callback)
```

Comment:
"HotplugCallback" instead maybe to clarify the use case for the callback?

Should clarify on what thread the callback is being called.